docs/source/AdministratorGuide/Resources/computingelements.rst

```diff
@@ -135,6 +135,32 @@ A commented example follows::
      }
    }
 
+
+HTCondor Computing Element
+@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+The configuration is similar to the CREAM example above.
+The following settings are recommended in::
+
+  Resources
+  {
+    Computing
+    {
+      HTCondorCE
+      {
+        # temporary directory
+        WorkingDirectory = /opt/dirac/condor
+	# if True, local scheduler must be installed and running
```

Comment:
Spacing is wrong here.

Reply:
I tried to fix it, but it's really hard writing blind in an editor.